firmware/rhizome.ino

```diff
@@ -76,19 +80,28 @@ void setup() {
 //    sprouts.push_back(new Ohmbrewer::HeatingElement( 2, new std::list<int>(1,5), 0, true, fakeTask ));
 
 // EX 4: A RIMS w/ a Pump to the Kettle
-    //TODO change to array
+
     // [ tube temp busPin ; tube heating powerPin ; tube heating controlPin ; mashtun temp busPin ; pump powerPin ; pump controlPin]
 
-    int tubePins[] = {0,1,2};
-    int tunPin = 3;
-    int pumpPins[] = {4,5};
-    //for(int i = 1; i < 5; i++) {rimsPins->push_back(i);}
+//    thermPins->push_front(0); //tempBus pin
+    thermPins->push_back(4); //control pin (relay pin)
+    thermPins->push_back(5); //power pin (switch pin)
+
+
+    int pumpPin = 3;
+//    int pumpPin2 = 4;
+//    thermPins->pop_front();
 
-    sprouts.push_back(new Ohmbrewer::RIMS( 1, tubePins, tunPin, pumpPins ));
-    ((Ohmbrewer::RIMS*)sprouts.front())->setState(true); // Turn everything on.
-    ((Ohmbrewer::RIMS*)sprouts.front())->getTube()->setTargetTemp(559);
+//    sprouts.push_back(new Ohmbrewer::RIMS( 1, thermPins, pumpPin ));          //DOSNT WORK
+    sprouts.push_back(new Ohmbrewer::Thermostat(1, thermPins));                 //WORKS
```

Comment:
If you change thermPins as I advise above (i.e. change it from a list pointer to simply a list), then this works:
```c++
    sprouts.push_back(new Ohmbrewer::RIMS( 1, &thermPins, pumpPin )); //DOES WORK
    sprouts.push_back(new Ohmbrewer::Thermostat(1, &thermPins));          //WORKS

```